storage-resize-images/README.md

```diff
@@ -35,7 +35,7 @@ To install an extension, your project must be on the [Blaze (pay as you go) plan
 - You will be charged a small amount (typically around $0.01/month) for the Firebase resources required by this extension (even if it is not used).
 - This extension uses other Firebase and Google Cloud Platform services, which have associated charges if you exceed the service’s free tier:
  - Cloud Storage
- - Cloud Functions (Node.js 10+ runtime. [See FAQs](https://firebase.google.com/support/faq#expandable-24))
+ - Cloud Functions (Node.js 12+ runtime. [See FAQs](https://firebase.google.com/support/faq#expandable-24))
```

Comment:
I was curious so I checked. New URL: https://firebase.google.com/support/faq#extensions-pricing

Reply:
Great spot Mike, reading that article suggest that this change probably isn't needed as part of the PR? As the document specifically notes v10+..?

Could always update that link as a separate PR?

Reply:
Mmm good point @dackers86 - probably leave 10+ runtime in the comment (since that is the version in the pricing document) but definitely point to the new URL

Reply:
Updated.